plugins/api-docs/src/components/OpenApiDefinitionWidget/OpenApiDefinitionWidget.tsx

```diff
@@ -28,6 +29,7 @@ const LazyOpenApiDefinition = React.lazy(() =>
 /** @public */
 export type OpenApiDefinitionWidgetProps = {
   definition: string;
+  requestInterceptor?: (req: Request) => Request | Promise<Request>;
```

Comment:
Alright let's just leave it as `any` 😅, we can add more strict types in the future but since `swagger-ui-react` doesn't give us anything better for now I think that's the way to go. Let's not declare a `Request` type, would like to avoid adding another `Request` type to the global namespace for imports.